dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -102,48 +102,47 @@ def execute(self, sql: str, *parameters: Any) -> None:
 
         # 0. generate a job name
         adapter_timer.start_timer("generateJobName")
```

Comment:
should this also be renamed for consistency

Reply:
Done. Missed it while refactoring.